src/app/core/components/breadcrumbs/breadcrumbs.component.ts

```diff
@@ -1,8 +1,12 @@
-import { Component } from '@angular/core';
+import { Component, Input } from '@angular/core';
+
+import { BreadcrumbsStep } from '@app/interfaces/helpers/breadcrumbs-step.interface';
 
 @Component({
   selector: 'app-breadcrumbs',
   templateUrl: './breadcrumbs.component.html',
   styleUrls: [ './breadcrumbs.component.scss' ],
 })
-export class BreadcrumbsComponent { }
+export class BreadcrumbsComponent {
+  @Input() steps: BreadcrumbsStep[];
```

Comment:
I think it will be great to move breadcrumbs presentation logic from app.component.ts to breadcrumbs.component.ts

Reply:
Agree, good idea